pkg/controller/datavolume/controller.go

```diff
@@ -87,20 +87,15 @@ type Event struct {
 
 type dataVolumeSyncResult struct {
 	dv      *cdiv1.DataVolume
+	dvCopy  *cdiv1.DataVolume
```

Comment:
maybe the naming can be improved here?  I assume one is the original and one is being mutated.  How to know which?

Reply:
Like the naming convention all over CDI,  the original is obj and the mutated is objCopy.

Reply:
Cpy may be fine when the variable exists in the scope of a function but not in a struct that is passed around without context

Reply:
Renamed to `dvMutated`.